_sass/components/_nav.scss

```diff
@@ -155,7 +155,7 @@
     margin-left: 1.5rem;
   }
 
-  .usa-nav__link span {
+  .usa-nav__link__inner {
```

Comment:
In [BEM naming](https://en.bem.info/methodology/naming-convention/#two-dashes-style), I wouldn't expect more than one `__`. I think we can call this:

```suggestion
  .usa-nav__link-inner {
```